hatchery/config.go

```diff
@@ -102,10 +106,15 @@ func LoadConfig(configFilePath string, loggerIn *log.Logger) (config *FullHatche
 		data.Logger.Printf("failed to load %v from cwd %v got - %v", configFilePath, cwd, err)
 		return data, err
 	}
-	data.Logger.Printf("loaded config: %v", string(plan))
 	data.ContainersMap = make(map[string]Container)
 	data.PayModelMap = make(map[string]PayModel)
-	_ = json.Unmarshal(plan, &data.Config)
+	_ = yaml.Unmarshal(plan, &data.Config)
+	configDebugTxt, _ := yaml.Marshal(data.Config)
```

Comment:
YAML is much more user friendly then JSON for config files.

Reply:
I don't disagree with that statement, but this is a breaking change because it is moving all the configs from json to yaml

So we either have to make this backward compatiable, or leave as-is but await for CTDS's timeline to move all of our config formats before we can merge this